amivapi/studydocs/endpoints.py

```diff
@@ -80,6 +81,11 @@
                 'maxlength': 30,
                 'nullable': True,
                 'type': 'string'
+            },
+            'coursesemester': {
+                    'maxlength': 5,
+                    'nullable': True,
+                    'type': 'string'
```

Comment:
Yes that was definitely my intent behind it. Was a bit unclear in my head as to how I would do it but Regex validation seems right.